src/aspire/denoising/__init__.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import numpy as np
+
+from aspire.source import ArrayImageSource
+from aspire.image import Image
+from aspire.io.starfile import save_star
+
+logger = logging.getLogger(__name__)
+
+
+class Denoiser:
+    """
+    Define a base class for denoising 2D images
+    """
+    def __init__(self, src):
+        """
+        Initialize an object for denoising 2D images from the image source
+
+        :param src: The source object of 2D images with metadata
+        """
+        self.src = src
+        self.dtype = src.dtype
```

Comment:
Needed?

Reply:
This is to make consistent with other classes. Removed in the new commit  4c22159.